proto/messages.proto

```diff
@@ -88,11 +89,9 @@ message Article {
 }
 
 message Mention {
-  // required string user_id = 1; // Deprecated it is optional now, use user_id = 3 instead
-  // required string user_name = 2; // Deprecated
-  optional string user_id = 3;
-  required int32 start = 4; // offset from beginning of the message counting in utf16 characters
-  required int32 end = 5; // offset from beginning of the message counting in utf16 characters
+  optional string user_id = 1;
```

Comment:
As we create a completely new `Mentions` message now we might want to consider making user_id non-optional and maybe using a `oneof` to distinguish between mentioning a single user and other potential types of  mentioning. E.g. something like:

```
message Mention {
  oneof mention_type {
       string user_id = 1;
       string mention_type = 2;
    }
 
  required int32 start = 2;
  required int32 end = 3
}
```

Instead of using a string to specify other mention types we could also introduce an enum.

Reply:
So maybe for now we can have `oneof` with only `user_id`? And add other fields if needed later

Reply:
@daehn I like the idea. @vytis ?

Reply:
@daehn updated with `oneof`